web/client/themes/default/ms2-theme.less

```diff
@@ -952,15 +952,65 @@ select.form-control option {
 	background-color: @ms2-color-primary !important;
 	color: @ms2-color-text-primary !important;
 }
-/* feauture grid */
+/* feature grid made by react-data-grid */
+#container-wrapped-component .react-grid-Header{
```

Comment:
.container-wrapped-component (same below)
use a better name like dockpanel-wrapped-component or something like this